source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
-- (Windows) Add Mattermost to Start menu, Taskbar, and System Tray.
-- (Windows/Mac) Deep link to the desktop app via ``mattermost://`` protocol if the app is already installed.
-- (Mac) Add Mattermost to the Dock.
-- (Linux) ``Desktop Entry`` for the application to more easily `integrate into a desktop environment <https://wiki.archlinux.org/index.php/Desktop_entries>`__.
+The Mattermost Desktop App is available for Windows, Mac, and Linux operating systems. The Desktop App supports all the features of the web experience, plus the following features:
 
-Below is a list of additional resources:
+- `Connect to multiple Mattermost servers <https://docs.mattermost.com/welcome/manage-desktop-app-server-connections.html>`__ from a single interface, and navigate between servers using keyboard shortcuts.
+- `Auto-start Mattermost <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ when a user logs into their machine.
+- `Add Mattermost <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ to the Windows Start menu, the Taskbar, the Dock, or the System Tray.
+- (Windows/Mac) `Deep link to the Desktop App <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ via the ``mattermost://`` protocol if the app is already installed.
+- (Linux) `Set up Desktop Entry <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__ for the application to more easily `integrate into a desktop environment <https://wiki.archlinux.org/index.php/Desktop_entries>`__.
 
-- `Guide for configuring your desktop app experience <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__
-- `Changelog <https://docs.mattermost.com/install/desktop-app-changelog.html>`__
+The following additional documentation resources are also available:
+
+- `Desktop App changelog <https://docs.mattermost.com/install/desktop-app-changelog.html>`__
+- `Minimum software requirements <https://docs.mattermost.com/install/software-hardware-requirements.html#desktop-apps>`__
+- `Configure your Desktop App experience <https://docs.mattermost.com/welcome/customize-desktop-app-experience.html>`__
 - `Source code <https://github.com/mattermost/desktop>`__
 - `Contributor’s guide <https://developers.mattermost.com/contribute/desktop>`__
 
-You can `download the desktop apps directly from our Downloads page <https://mattermost.com/download/>`__. You may also use the following installation guides for Windows, Mac, and Linux.
+Install the Mattermost Desktop App
+----------------------------------
+
+You can `download the Desktop App directly from our Downloads page <https://mattermost.com/download/>`__. You may also use the following installation guides for Linux, Mac, and Windows.
 
 .. include:: ../upgrade/upgrading-to-v60.rst
+  :start-after: :nosearch:
+
+.. tabs::
+
+  .. tab:: Linux
+
+    **Generic Linux package** 
+
+    1. Download the latest version of the Mattermost desktop app:
```

Comment:
Need updated download URLs

Reply:
The ``5.0.4`` can be replaced with ``5.1.0`` in the URLs.
I have a task as part of the release checklists to update the URLs before a release - I'm 0/5 if you update the URLs as part of this PR, or I can update in a separate PR. Up to you!

Reply:
There may be a URL that takes users to the latest available version. That would be ideal and would eliminate the need to maintain these URLs each release. @devinbinnie - would you be open to investigating this further?

Reply:
I might be mistaken here, but the ones I may have been thinking of are for the Rainforest builds, which are currently pointing to daily, which isn't ideal for our docs.

Other than that I don't know where I would be able to investigate since I don't control those URLs. @amyblais If we don't have this who would be best to set up a link like I described? I think it would save some of @cwarnermm's time so that she doesn't have to update everytime we release an app.

Reply:
@devinbinnie Normally I have been updating the download links for each release as part of the release checklists. @cwarnermm, let me know if you'd like me to help with this for v5.1.

I'm not sure if @pfltdv knows if we have a "latest" download link for desktop app, or maybe we can create a ticket to add it for future.

Reply:
@amyblais - Very happy for your help for this upcoming v6.7 release. Longer term, it would be ideal to have a "latest" link which would reduce maintenance each (major) release.

Reply:
As per my comment below the latest link for which package? Currently its the tarball with "other options" being found out on GitHub. Until we extend the UX on the /downloads page we would need to follow this approach.

@devinbinnie does the tarball install method also support the auto-updater?

Reply:
@amyblais We do not have latest download link for desktop app.